pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+members will see reports within these categories by default.
+
+</div>
+
+<div class="pro-task" markdown="1" id="defining-area-interest">
+
+### Defining the area of interest
+
+<span class="pro-task__permissions">Permissions required: none</span>
+
+If required, you can also use the controls to the right of the screen to zoom in or out, or pan across
+the map. As you’ll see, the list of reports always reflects the area you are viewing on the map, so
+this is another way of refining your view.
+
+</div>
+
+<div class="pro-task" markdown="1" id="view-report-specific-area">
+
+### Viewing a report in a specific area
+
+If you already know the street name or postcode of the location you’d like to view, you may find it
+quicker to use the search form on the front page.
+
+</div>
+
+<div class="pro-task" markdown="1" id="filtering-sorting-reports">
+
+### Filtering and sorting reports
+
+<span class="pro-task__permissions">Permissions required: none</span>
+
+<img alt="The drop-downs at the top of the report list allow you to filter which reports you see" src="/assets/img/pro-user-guide/filtering-sorting-reports.png" class="pro-screenshot" />
+
+The drop-downs at the top of the report list allow you to filter which reports you see, by state (e.g.
+‘Fixed’ or ‘In progress’) and category (e.g. Potholes or Graffiti). You may also sort them by date
+created, how recently they have been updated, or number of updates.
+
+</div>
+
+<div class="pro-task" markdown="1" id="view-single-report">
+
+### Viewing a single report
+
+<span class="pro-task__permissions">Permissions required: none</span>
+
+Click on one of the issues in the list, or on a map pin, to view the full report.
+See '[Dealing with undesirable content](#deal-undesirable-content)‘ below for how to moderate or hide a report.
+
+</div>
+
+<div class="pro-task" markdown="1" id="search-report">
+
+### Searching for a report
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff</span>
+
+
+Click on ‘admin’ in the top menu bar.
+
+There are a number of different options for using the Search Reports box:
+
+- Enter any word contained in the required report’s title or body
+- Search by the report-maker’s name (or part of the name), their email address or phone
+    number (where given).
+- Search by the report’s ID number if it’s known. Putting ‘id:’ directly before this number in
+    the search box will speed up the return of results.
+
+<img alt="There are a number of different options for using the Search Reports box" src="/assets/img/pro-user-guide/search-report.png" class="pro-screenshot" />
+
+The search will return a list of all reports that fulfil your parameters. Click on the report’s ID
+number to see it in a new browser window; or click on the word ‘Edit’ to gain access the tools that
+(depending on the staff user’s permissions) allow editing of reports, prioritisation, changing a
```

Comment:
Some note here that this is full admin editing, not inspector front-end editing (which could also do all of that).

Reply:
This might need a disambiguation of its own, nearer the beginning, something along the lines of:
>There are two editing interfaces:
>*Front-end editing*: on-page report editing, accessible via the FixMyStreet report pages when the staff user is logged in
>*Full admin editing*: back-end reporting, accessible only to those staff with admin rights, and allowing actions such as managing user permissions, as well as providing a route to the on-page reporting functions.

(TBH I'm quite vague about this myself, so do check whether that's true)

Then this section would presumably have to read:
>In the full admin interface, click on the report's ID....

Reply:
For now I have gone with “The search will return a list of all reports that fulfil your parameters. Depending upon the staff user’s permissions, you can then either click on the report’s ID number to open the report in a new browser window and access the moderation/ editing/ inspection facilities there; or click on the word ‘Edit’ to access the full report admin editing page.”